frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/PreviewPane/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { match } from "ts-pattern";
+
+type Theme = "light" | "dark";
+export function getCheckerBoardDataUri(theme: Theme) {
```

Comment:
I was thinking if we can use `PreviewBackgroundType` as the `theme` here, but on second thought I think this is better as the checkerboard pattern can be used anywhere else.

Reply:
on second thought, I might buy using `PreviewBackgroundType` for this. I don't intend `getCheckerBoardDataUri` to be used anywhere else except for static embed preview anyway.

Reply:
Fixed in https://github.com/metabase/metabase/pull/45134/commits/a171252f8236a8aa738c08a8e1a7d90170839174